tests/unit/presigned_get_object_test.py

```
@@ -63,3 +65,35 @@ def test_can_include_response_headers(self):
+            endpoint='localhost:9000',
+            access_key='my_access_key',
+            secret_key='my_secret_key',
+        )
+        url = client.presigned_get_object(
+            bucket_name='mybucket',
+            object_name='myfile.pdf',
+            region='us-west-2',
+        )
+        self.assertIn('%2Fus-west-2%2Fs3%2Faws4_request', url)
+
+    def test_region_conflicts_with_client_region(self):
```

Pylint (4.0.6)

[convention] Method name "test_region_conflicts_with_client_region" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern

(C0103)